packages/ui/src/components/va-button-toggle/VaButtonToggle.demo.vue

```diff
@@ -249,6 +266,16 @@ export default {
         { label: 'Two', value: 'two' },
         { label: 'Three', value: 'three' },
       ],
+      optionsWithIcons: [
+        { label: 'One', value: 'one', icon: 'home' },
+        { label: 'Two', value: 'two', iconRight: 'home' },
+        { label: 'Three', value: 'three', icon: 'home', iconRight: 'home' },
```

Comment:
I personally prefer if this issue will have slot based fix. But it looks nice anyway.

Do we also want to have?: 
```vue
<VaButton label="Click me" />
```

Then options will look pretty as VaButton props. So we can pass any prop in options:
```
options: { type: Array as PropType<(VaButtonProps & { value: any })[]> }
```

This way we make a beautiful IDE support.